tests/src/server/CMakeLists.txt

```
@@ -15,6 +15,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}
 # Tests:
 
 set(TESTS
+  test_qgsserver_interface.cpp
```

```suggestion
  testqgsserverinterface.cpp
```

I think the naming convention for cpp tests has no `_`

---

Indeed you're right :+1: